src/test/suite/fixtures/recorded/languages/cpp/chuckValue.yml

```diff
@@ -0,0 +1,105 @@
+languageId: cpp
```

Comment:
Killing 10 birds with one stone I see 😊

Reply:
Yes I didn't feel like adding 30 tests for this :D